pkg/util/util.go

```diff
@@ -9,6 +9,10 @@ import (
 	yaml "gopkg.in/yaml.v2"
 )
 
+var (
+	BuildVersion, BuildDate, GitTag string
```

Comment:
If you don't need the `VERSION` file for other reasons, you could just use the `GitTag` internally, since your build pipeline keeps it in sync already.